src/Microsoft.ML.OnnxTransformer/OnnxTransform.cs

```diff
@@ -96,6 +96,9 @@ internal sealed class Options : TransformInputBase
 
             [Argument(ArgumentType.AtMostOnce, HelpText = "Controls the number of threads to use to run the model.", SortOrder = 8)]
             public int? IntraOpNumThreads = null;
+
+            // No argument cause it can't be used via cmd
+            public Stream ModelBytes = null;
```

Comment:
How is the cmd scenario looks like

Reply:
We don't support the stream for cmd so no scenario there for it. Still have to use the string/filepath for cmd.